realistic_test.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+Copyright (c) 2024 braintec AG (https://braintec.com)
+All Rights Reserved
+Licensed under the Odoo Proprietary License v1.0 (OPL).
+See LICENSE file for full licensing details.
+-->
```

Comment:
**[question]** Why this license? Licenses in all files or in none, right?

Reply:
I just wanted to have a test file as close to a an actual view file we could have. But yes, maybe having the license there bound us legally to some issues. I could replace it with 2 fake license lines, what do you think about it?

Reply:
No clue. Maybe HOD has an idea? @BT-rmartin

Reply:
@BT-dherreros yes, let's go with 2 fake lines